src/initFirebaseClientSDK.js

```diff
@@ -11,6 +11,10 @@ export default function initFirebaseClientSDK() {
       )
     }
     firebase.initializeApp(firebaseClientInitConfig)
```

Comment:
The `tenantId` we added to `config.firebaseClientInitConfig` isn't a property for the Firebase app config, so let's break it out:
```js
const { tenantId, ...firebaseConfig } = firebaseClientInitConfig
```
and use `firebaseConfig` to init.

Please also add test coverage for this behavior.